bee-message/src/output/alias.rs

```diff
@@ -171,6 +195,8 @@ pub struct AliasOutput {
     unlock_conditions: UnlockConditions,
     //
     feature_blocks: FeatureBlocks,
+    //
```

Comment:
What's up with the blank comments? Are they supposed to be done later in another PR?

Reply:
Yes, once the TIPs are stable enough (they probably are now), I'll do one big PR with all the docs of bee-message